catalyst-gateway/bin/src/service/api/cardano/cip36/mod.rs

```diff
@@ -1,13 +1,12 @@
 //! CIP36 Registration Endpoints
 
-use poem::http::HeaderMap;
 use poem_openapi::{param::Query, payload::Json, OpenApi};
 use response::Cip36RegistrationUnprocessableContent;
 
-use self::{cardano::slot_no::SlotNo, common::auth::none::NoAuthorization};
-use super::Ed25519HexEncodedPublicKey;
+use self::cardano::slot_no::SlotNo;
 use crate::service::common::{
     self,
+    auth::none_or_api_key::NoneOrApiKey,
```

Comment:
The API key was specifically hidden on this endpoint so it would not appear in the API, as was the "ALL" option.
They are internal only options not to be advertised through the public API.

Reply:
I reverted these changes.